### PR TITLE
Revert "add meta properties to xform"

### PR DESCRIPTION
**Pull request by emord**
Reverts dimagi/commcare-hq#19368

broke the biuld

### CODE REVIEW

**Comment by snopoke** [comment]
@emord @sravfeyn was there a plan to re-do this work? Right now it's half done since the SQL models fields are there but are not being populated.

File: corehq/ex-submodules/couchforms/models.py [hunk 1]
```diff
@@ -226,20 +226,20 @@ def metadata(self):
 
     @property
     def time_start(self):
-        return self.metadata.timeStart
+        # Will be addressed in https://github.com/dimagi/commcare-hq/pull/19391/
```
Comment (snopoke) [inline corehq/ex-submodules/couchforms/models.py]: @emord this doesn't seem to be the right PR - any idea what the correct URL is?

**Comment by emord** [comment]
Its possible that https://github.com/dimagi/commcare-hq/pull/19847 should have included this? Its using the same branch name and says its a version of the old PR, but does not include these files

**Comment by sravfeyn** [comment]
I don't quite remember what we decided. 

We can revisit it if needed, I have opened a revert of this [PR](https://github.com/dimagi/commcare-hq/pull/24904) and will watch out for the bug that broke the build to see if we can get this in.

**Pull request merged**